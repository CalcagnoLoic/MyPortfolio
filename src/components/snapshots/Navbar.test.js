import renderer from "react-test-renderer";
import Navbar from "../Navbar";

it("render correctly Navbar component", () => {
    const componentNavbar = renderer.create(<Navbar />).toJSON();
    expect(componentNavbar).toMatchSnapshot();
});
