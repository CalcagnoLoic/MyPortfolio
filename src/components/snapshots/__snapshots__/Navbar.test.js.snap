// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`render correctly Navbar component 1`] = `
<nav
  className="flex flex-row py-6 md:py-2 pl-5 pr-5 lg:pl-28 lg:pr-28 justify-between xl:flex-row  bg-main-color z-20 md:fixed md:w-full"
>
  <div
    className="w-full flex justify-between"
  >
    <div
      className="flex self-center"
    >
      <p
        className="text-[20px] md:text-3xl xl:text-3xl sm:text-2xl text-white xl:text-left break-words"
        id="top"
      >
        Portfolio | Calcagno Lo√Øc
      </p>
    </div>
    <div
      className="hover:cursor-pointer md:hidden z-20 self-center"
      onClick={[Function]}
    >
      <img
        alt="Logo menu hamburger"
        className=""
        src="icon-hamburger.svg"
      />
      <img
        alt="Logo menu closed"
        className="hidden fixed right-5 top-8"
        src="icon-close.svg"
      />
    </div>
  </div>
  <nav
    className="fixed top-[-100vh] left-0 right-0 h-screen bg-main-color transition-all duration-700 md:hidden z-10 "
    onClick={[Function]}
  >
    <div
      className="flex flex-col gap-16 justify-center mt-5 mb-5 lg:my-5 md:my-5 sm:my-5 md:flex md:gap-10 px-5 items-center h-full"
    >
      <a
        className="text-white text-3xl hover:cursor-pointer hover:italic hover:underline hover:underline-offset-[10px] font-[bellefair]"
        href="#projects"
      >
        Projects
      </a>
      <a
        className="text-white text-3xl hover:cursor-pointer hover:italic hover:underline hover:underline-offset-[10px] font-[bellefair]"
        href="#skills"
      >
        Skills
      </a>
      <a
        className="text-white text-3xl hover:cursor-pointer hover:italic hover:underline hover:underline-offset-[10px] font-[bellefair]"
        href="#contact"
      >
        Contact
      </a>
    </div>
  </nav>
  <nav
    className="hidden md:flex md:gap-12"
  >
    <div
      className="flex gap-16 self-center mt-5 mb-5 lg:my-5 md:my-5 sm:my-5 md:flex md:gap-10"
    >
      <a
        className="text-white text-[18px] xl:text-xl md:text-xl hover:cursor-pointer hover:italic hover:underline hover:underline-offset-[10px] font-[bellefair]"
        href="#projects"
      >
        Projects
      </a>
      <a
        className="text-white text-[18px] xl:text-xl md:text-xl hover:cursor-pointer hover:italic hover:underline hover:underline-offset-[10px] font-[bellefair]"
        href="#skills"
      >
        Skills
      </a>
      <a
        className="text-white text-[18px] xl:text-xl md:text-xl hover:cursor-pointer hover:italic hover:underline hover:underline-offset-[10px] font-[bellefair]"
        href="#contact"
      >
        Contact
      </a>
    </div>
  </nav>
</nav>
`;
